src/plugins/graph/adornments/plotted-function/plotted-function-adornment-types.ts

```diff
@@ -0,0 +1,11 @@
+export const kPlottedFunctionClass = "plotted-function";
+export const kPlottedFunctionType = "Plotted Function";
+export const kPlottedFunctionPrefix = "ADRN";
+export const kPlottedFunctionLabelKey = "DG.Inspector.graphPlottedFunction";
+export const kPlottedFunctionUndoAddKey = "DG.Undo.graph.showPlotFunction";
+export const kPlottedFunctionRedoAddKey = "DG.Redo.graph.showPlotFunction";
+export const kPlottedFunctionUndoRemoveKey = "DG.Undo.graph.hidePlotFunction";
+export const kPlottedFunctionRedoRemoveKey = "DG.Redo.graph.hidePlotFunction";
+export const kPlottedFunctionValueTitleKey = "DG.PlottedFunction.valueTitle";
```

Comment:
These look like they are wanting to key into a file of localization strings, which I don't think we have.

Reply:
Most of these were unused so I just removed them.

I kept `kPlottedFunctionLabelKey` because the connecting lines adornment has something similar.